Mamajin/Event-Reservation
# Feature/decouple settings

## Description: 
This pull request decouples the Django settings to use environment variables stored in a .env file, improving security and configurability across different environments. It ensures sensitive data like SECRET_KEY, DATABASE_URL, and other configuration values are not directly hard-coded in the settings.py file.

## Changes:

- Integrated python-decouple for environment variable management.
- Updated settings.py to load configurations from the .env file.
- Added sample.env as a template for required environment variables.
- Added a README section on how to set up the .env file for development.

## How to Test:

- Copy sample.env to .env and populate it with appropriate values.
- Run python manage.py runserver to start the server.
- Verify that the application works as expected without any configuration errors.

## Checklist:

 - I have run the code and it functions as expected.
 - I have updated the documentation (if necessary).
 - The code follows the project's style guidelines.
 - 
## Additional Information: 
This change improves the security of sensitive information and makes it easier to switch between different configurations for development, testing, and production. Developers should ensure they have their .env file correctly configured when pulling this change.
